### ISSUE TITLE
Doesn't apply to .svelte files

### ISSUE THREAD

**Issue by dancherb** [outofsync42/inline-css-color#2]
Hi - thanks for the extension, works great on .html files but the reason I sought something like this out was to use it in Svelte - where the files take a format very similar to traditional .html files. Would it be difficult for the extension to also apply to .svelte files?

**Comment by outofsync42** [comment]
I've never worked on svelte files before but I can certainly take a look and try to update the parser. Ill let you know.

**Comment by dancherb** [comment]
Awesome, thanks. I've attached a demo of a component, Component.svelte below - this users Javascript within inline styles. There's also a few standard VSCode extensions on Svelte that most developers will have installed. 

(No pressure of course! And Svelte is awesome either way, maybe you'd enjoy checking it out)

```
<script>
	export let bold;
	export let italic;
	export let underline;
	
	export let white;
	export let center;
	export let right;
	
	export let inline
	
	export let style

	const font = 'Calibri, sans-serif';
	const fontSize = 14;
	
	console.log('style =', style)
	
	const testFunction = () => {
		console.log('hi')
	}
</script>

<svelte:head>
	<title>My website - home</title>
</svelte:head>

<div
	style='
		font-size: {fontSize}px;

		font-style: {italic ? 'italic' 
		: 'normal'};

		color: {white ? 'white' 
		: '#475467'};

		text-align: {center ? 'center' 
		: right ? 'right' 
		: 'left'};

		text-decoration-line: {underline ? 'underline' 
		: undefined};

		height: 200px;
		width: 200px;
		
		white-space: pre-line;

		font-family: {font};

		font-weight: {bold ? '700' : '400'};

		{inline ? `display: 'inline';` : ``}

		{style}
	'
>
	<slot/>
</div>

<style>
	div {
		box-sizing: border-box;
		white-space: pre-line;
	}
</style>
```

**Comment by outofsync42** [comment]
That's very helpful. Thank you. Working on the parser now. Should have a release in a day or two

**Comment by outofsync42** [comment]
Released Version 2.1.0 with added support for svelte. :) Let me know if it works as desired. 

One thing to note I decided not to parse inside strings on the value side of the ternary operations as that would have been a lot of work to distinguish a full css property versus just a value.  

For example, this would have been easy to parse and determine that the numbers should have been colored
`font-weight: {bold ? '700' : '400'};`

This would have been hard to parse as prints a complete css property value pair at an unexpected position that I wouldn't know ahead of time needs to run the beginning of the rule set on. Rules are based on positions of punctuation and order of operation.
`{inline ? `display: 'inline';` : ``}`
